_posts/2019-03-07-the-importance-of-sprint-retrospectives.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "The Importance of Sprint Retrospectives"
+date: 2019-03-07 15:00:00
+categories: ["agile", "project-management"]
+author: "emily"
+---
+
+Sprint Retrospectives are a crucial part of our project management strategy at [Ombu Labs](https://www.ombulabs.com). Because we are a remote and international team of software developers, we miss out on the “water-cooler” conversations and small talk that would normally occur between employees in a typical office environment.
+
+<!--more-->
+
+Though it may seem small, those “hey, what are you working on today?” and “i’m having trouble with a client” conversations help keep the team up to speed.
+
+It is important for project team's to make time to check in and discuss how the project is going. For this we have the [**Sprint Retrospective**](https://www.scrum.org/resources/what-is-a-sprint-retrospective).
+
+We like to **schedule a Sprint Retrospective for each client project every two weeks**, and invite all team members working on that project to join in.
+
+During the Sprint Retrospective, we have each team member discuss the following questions:
+
+1. What went well in the sprint
+2. What didn’t go well in the sprint
+3. Happiness level with the project (1 to 5)
+4. What can be improved
+
+In the call **it is important to give each team member time to share their answers in depth** with the rest of the team. We then like to give other team members time to respond and ask questions.
+
+If a team member shares that they are having an issue or believes that something is not going well in the project, try to encourage them to share a suggestion for how it can be improved. How can we do better?
```

Comment:
@ercohen14 One more comment, that last question ("How can we do better?") doesn't fit that good  there to me. (maybe it's just me)